Gordon360/ApiControllers/HousingController.cs

```diff
@@ -78,12 +82,28 @@ public IHttpActionResult AddHousingAdmin(string id)
         /// <returns></returns>
         [HttpDelete]
         [Route("admin/{id}")]
+        [StateYourBusiness(operation = Operation.DELETE, resource = Resource.HOUSING_ADMIN)]
         public IHttpActionResult RemoveHousingAdmin(string id)
         {
             bool result = _housingService.RemoveHousingAdmin(id);
             return Ok(result);
         }
 
+        /// <summary>
+        /// Delete an application (and consequently all rows that reference it)
+        /// </summary>
+        /// <param name="applicationID"> The id of the application to remove </param>
+        /// <returns></returns>
+        [HttpDelete]
+        [Route("apartment/applications/{applicationID}")]
+        // StateYourBusiness
```

Comment:
This route really needs authorization via StateYourBusiness.

Reply:
#559 has this and the submit's